kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/StatisticsCollector.java

```diff
@@ -43,27 +43,18 @@ public void calculateAdvStatistics(Statistics statistics) {
         postsQuantity = statistics.postsCount();
         commentsQuantity = statistics.commentsCount();
 
-        if (usersQuantity > 0) {
+        if (usersQuantity == 0) {
+            avgQuantityPostsOnUser = Double.NaN;
```

Comment:
ja bym zwracał jednak zero :) tak intuicyjnie to czuję :) wiem, że wcześniej w zadaniu zwracaliśmy NaN. Z NaN jest problem w Javie 11, ponieważ tam nie ma już takiego czegoś. 
No i po co ci pola klasy? Pola klasy mówią o jakiejś własności obiektu/klasy. Tutaj dostajemy obiekt Statistics i za jego pomocą obliczamy odpowiednie dane. Na przykład return statistics.postsCount() / usersNumber() (jeśli usersNumber() != 0) w przypadku średniej ilości postów na użytkownika itp. usersNumber() to poprostu statistics.size() ...